receiver/apachereceiver/metadata.yaml

```diff
@@ -174,3 +181,11 @@ metrics:
       monotonic: false
       aggregation_temporality: cumulative
     attributes: [scoreboard_state]
+  apache.connections.async:
+    enabled: true
```

Comment:
this is a breaking change. Please refer to https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md#changing-the-emitted-metrics

You can add this metric as optional and turn it as default as a breaking change on in a separate PR, or change this changelog to indicate this is a breaking change.

Reply:
Thanks for pointing that out. I have changed the type in the changelog to reflect this. Hope it's okay now.